src/common/helpers/itemframes.ts

```diff
@@ -0,0 +1,54 @@
+import { Block, BlockFace } from 'org.bukkit.block';
+import { EntityType, ItemFrame } from 'org.bukkit.entity';
+import { ItemStack } from 'org.bukkit.inventory';
+
+/**
+ * Summons an invisible item frame
+ * @param block The cube to which the item frame is attached
+ * @param face The Blockface to which the item frame is attached
+ * @param item The itemstack to be placed
+ * @returns The summoned item frame
+ */
+export function summonInvisibleItemFrame(
```

Comment:
```suggestion
export function spawnInvisibleItemFrame(
```
Bukkit usually uses 'spawn', even though the Vanilla command is called /summon.